docs/container-contract.md

```diff
@@ -4,32 +4,31 @@ linkTitle: "Container Contracts"
 weight: 8
 ---
 -->
-# Container Contract
+# Container Contracts
```

Comment:
I don't think we have more than one? `Contract` sounds more correct to me.

Reply:
Fixed. It was originally a plural and I thought maybe this doc would be expanded to add more. I'm not entirely sure what a container contract is myself. :)